UNIFY-673-supportFile-updates (#4280)
cypress-io/cypress-documentation: content/partials/support-file-configuration.md

A great place to put this configuration is in the
[supportFile](/guides/core-concepts/writing-and-organizing-tests#Support-file),

> since it is loaded before any test files are evaluated.

Reviewer: ```suggestion
since it is loaded before each spec files is evaluated.
```
